src/modules/governance/ProposalListItem.tsx

```diff
@@ -18,6 +19,35 @@ export function ProposalListItem({
   const { nayPercent, yaePercent, nayVotes, yaeVotes, quorumReached, diffReached } =
     formatProposal(proposal);
 
+  const delayedBridgeExecutors = [
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_ARBITRUM,
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_OPTIMISM,
+    AaveGovernanceV2.CROSSCHAIN_FORWARDER_POLYGON,
+  ];
+
+  let proposalCrosschainBridge = false;
+
+  if (proposal.targets && proposal.targets.length > 0) {
+    const hasDelayedExecutor = proposal.targets.filter((address) =>
+      delayedBridgeExecutors.includes(address)
+    );
+    if (hasDelayedExecutor.length > 0) {
+      // exectutionTimeStamp = executionTime + 172800; // Adds time for cross bridge execution
+      proposalCrosschainBridge = true;
+    }
+  }
+
+  // Note: We assume that the proposal will be executed two days later and add 3 hour buffer
+  const twoDayDelay = 172800;
+  const threeExtraHours = 10800;
```

Comment:
Instead of picking a buffer which may be over or under, why not remove the buffer then setup Gelato executors to instantly execute the proposals on each L2 🤔